### ISSUE TITLE
Missing labels / accessible names for input and select elements

### ISSUE THREAD

**Issue by aviannamiller** [omeka-s-modules/Collecting#93]
Backend form inputs are missing labels and selects are missing accessible names, and on published forms, inputs are also missing labels.

Seemingly across the board, labels are indeed present, but the associated inputs are missing the proper IDs to link them (looks like this information is included in `name` attributes, instead).
